tools/Linux/firewalld-services/kodi-http.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<service>
+  <short>Kodi web interface</short>
+  <description>Kodi is a free cross-platform media-player jukebox and entertainment hub.  Enable this option to remotely control Kodi via its web interface.</description>
+  <port protocol="tcp" port="8080"/>
```

Comment:
Only problem I can think of is that you can edit this port inside of kodi and it won't match this anymore. Not sure if you can do for the others too, but this one is quiet prominent in the settings. 

Frustratingly the only thing I can think of we can do, is point that out in the description.

Reply:
I suppose the same thing is true of pretty much every service that `firewalld` knows about, since most services are configurable in this way, but I don't think it's a problem.  If someone's changed the port, they should know what port they've set it to, and should be able to tell that to the firewall.  Teaching `firewalld` about the default ports mainly helps users who don't already know what the default port numbers are.